PointOfInteraction/SmartSpot/doc/spec.md

```diff
@@ -0,0 +1,70 @@
+# SmartSpot
+
+## Description
+
+Smart Spots are devices which provide the technology to allow users to interact (suggestions mailbox, co-creation, …) or to obtain extra information (tourism, infotainment, …). The data model contains resources to configure the interaction service such as the broadcasted URL (probably will be a shortened url), the interval between broadcasts (frequency), the availability of the service, transmission power in function of the area to cover, etc.
+
+In addition to the presented data model, this entity inherits the Device data model. This means that by hierarchy, the SmartSpot entity is a child or specialization of a Device.
+
+These resources (except references to other entities) are mapped from a real device. For that reason, the changes in any of the values of this entity will modify the value in the real device.
+
+## Data Model
+
++ `id` : Unique identifier. 
+
++ `type` : Entity type. It must be equal to `SmartSpot`.
+
++ `announcedUrl` : URL broadcasted by the device.
+    + Attribute type: [URL](https://schema.org/URL)
+    + Mandatory    
+
++ `signalStrenght` : Signal strength to adjust the announcement range.
+    + Attribute type: [Text](https://schema.org/Text)
+    + Allowed values: "lowest", "medium" or "highest". 
+    + Optional    
+
++ `bluetoothChannel` : Bluetooth channels where to transmit the announcement.
+    + Attribute type: [Text](https://schema.org/Text)
+    + Allowed values: "37", "38", "39", "37-38", "38-39", "37-39" or "37-38-39".
+    + Optional  
+
++ `areaRadius` : Spot coverage area in meters.
```

Comment:
what about renaming this property to `areaCovered` ? Radius of the spot coverage area. 

When we are talking about magnitudes we usually declare the default unit separately, please check some of the properties at

http://fiware-datamodels.readthedocs.io/en/latest/Weather/WeatherObserved/doc/spec/index.html